From: Tyler Hicks <tyhicks@canonical.com>
Date: Mon, 29 Jan 2018 12:43:15 +0000
Subject: Resolve includes for hardware specific accesses directory

When using the #include directive in an AppArmor profile, the parser should
search /usr/share/apparmor in addition to /etc/apparmor.d.

This is needed because Ubuntu places hardware specific access rules in
/usr/share/apparmor/hardware.

Note that the addition of this search patch may result in namespace collisions
among the two include directories.

Forwarded: not-needed
---
 parser/parser.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/parser/parser.conf b/parser/parser.conf
index 641cf1d..8db2626 100644
--- a/parser/parser.conf
+++ b/parser/parser.conf
@@ -21,6 +21,7 @@
 #Include /etc/apparmor.d/
 # or
 #Include /usr/share/apparmor
+Include /usr/share/apparmor
 
 
 ## Set location of apparmor filesystem
